response.h

```diff
@@ -24,12 +24,15 @@ class Response {
     NOT_FOUND = 404,
     FOUND = 302
   };
-  
+
   void SetStatus(const ResponseCode response_code);
```

Comment:
We could ask the Assignment 7 team to practice this rename in a foreign codebase. 😉